code/modules/unit_tests/merge_type.dm

```diff
@@ -1,5 +1,6 @@
 /datum/unit_test/merge_type/Run()
-	var/list/paths = subtypesof(/obj/item/stack) - /obj/item/stack/sheet - /obj/item/stack/sheet/mineral
+	var/list/paths = subtypesof(/obj/item/stack) - /obj/item/stack/sheet - /obj/item/stack/sheet/mineral - /obj/item/stack/ore
+	- /obj/item/stack/spacecash - /obj/item/stack/license_plates - /obj/item/stack/tile/mineral - /obj/item/stack/tile
```

Comment:
```suggestion
	var/list/blacklist = list(
		/obj/item/stack/sheet,
		/obj/item/stack/sheet/mineral,
		/obj/item/stack/ore,
		/obj/item/stack/spacecash,
		/obj/item/stack/license_plates,
		/obj/item/stack/tile/mineral,
		/obj/item/stack/tile,
	)
	var/list/paths = subtypesof(/obj/item/stack) - blacklist
```

Reads better and is easier to add onto.

Reply:
Implemented just as you added review, heh.